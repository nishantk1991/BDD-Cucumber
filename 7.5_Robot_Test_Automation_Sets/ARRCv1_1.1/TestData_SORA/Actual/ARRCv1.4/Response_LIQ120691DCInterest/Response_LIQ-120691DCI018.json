{
  "comment": " The most recent published SORA RFR rate is available as-of: 2020-05-22. Computed time : 2021-11-23T12:04:29+00:00. Computed time is the time stamp when the calculation was completed.",
  "lastUpdatedFromSource": "2021-10-29T00:00:00+00:00",
  "legalDisclaimer": "SORA-related statistics, the 1-month, 3-month and 6-month Compounded SORA rates, and the SORA Index, are provided on an as is basis, without warranties of any kind. MAS shall not be liable for any damage or loss, which may arise from reliance on such data obtained from the MAS website. Please refer to the Terms of Use (https://www.mas.gov.sg/terms-of-use) on accessing and using any part of the MAS website.",
  "interestCalculatorCriteria": {
    "initialPrincipalBalance": 10000000,
    "interestPeriodStartDate": "2020-05-01",
    "interestPeriodEndDate": "2020-06-05",
    "riskFreeRate": "SORA",
    "riskFreeRateFloor": 0.05,
    "lookBackDays": 5,
    "lockoutDays": 0,
    "observationPeriodShift": true,
    "spread": 0,
    "spreadAdjustment": -1,
    "dayCountConvention": "ACT/360",
    "rateRoundingDecimalPoints": 11,
    "paymentRounding": "NEAREST",
    "principalAmountChanges": [
      {
        "date": "2020-05-15",
        "principalAmountChange": 150000
      },
      {
        "date": "2020-05-16",
        "principalAmountChange": -250000
      },
      {
        "date": "2020-05-17",
        "principalAmountChange": -9900000
      }
    ],
    "showDailyDetails": true,
    "calculateTillDate": "2020-05-17"
  },
  "currencyCode": "SGD",
  "interestSummary": [
    {
      "interestPeriodDate": "2020-05-17",
      "interestPeriodDays": 3,
      "observationPeriodDate": "2020-05-07",
      "observationPeriodDays": 4,
      "principalBalance": 0.0,
      "publishedRiskFreeRate": 0,
      "rateApplied": 0.05,
      "riskFreeRateFloorApplied": true,
      "compoundingFactor": 0.909000880856,
      "dailyCompoundedRate": 0.04545004404,
      "allInRate": -0.95454995596,
      "interestAmount": 0.0,
      "cummulativeInterestAmount": -4172.98
    },
    {
      "interestPeriodDate": "2020-05-16",
      "interestPeriodDays": 3,
      "observationPeriodDate": "2020-05-07",
      "observationPeriodDays": 4,
      "principalBalance": 9900000.0,
      "publishedRiskFreeRate": 0,
      "rateApplied": 0.05,
      "riskFreeRateFloorApplied": true,
      "compoundingFactor": 0.909000880856,
      "dailyCompoundedRate": 0.04545004404,
      "allInRate": -0.95454995596,
      "interestAmount": -262.5,
      "cummulativeInterestAmount": -4172.98
    },
    {
      "interestPeriodDate": "2020-05-15",
      "interestPeriodDays": 3,
      "observationPeriodDate": "2020-05-07",
      "observationPeriodDays": 4,
      "principalBalance": 10150000.0,
      "publishedRiskFreeRate": 0,
      "rateApplied": 0.05,
      "riskFreeRateFloorApplied": true,
      "compoundingFactor": 0.909000880856,
      "dailyCompoundedRate": 0.04545004404,
      "allInRate": -0.95454995596,
      "interestAmount": -269.13,
      "cummulativeInterestAmount": -3910.47
    },
    {
      "interestPeriodDate": "2020-05-14",
      "interestPeriodDays": 1,
      "observationPeriodDate": "2020-05-06",
      "observationPeriodDays": 1,
      "principalBalance": 10000000.0,
      "publishedRiskFreeRate": 0.0111,
      "rateApplied": 0.05,
      "riskFreeRateFloorApplied": true,
      "compoundingFactor": 0.977265955303,
      "dailyCompoundedRate": 0.04886329777,
      "allInRate": -0.95113670223,
      "interestAmount": -264.2,
      "cummulativeInterestAmount": -3641.34
    },
    {
      "interestPeriodDate": "2020-05-13",
      "interestPeriodDays": 1,
      "observationPeriodDate": "2020-05-05",
      "observationPeriodDays": 1,
      "principalBalance": 10000000.0,
      "publishedRiskFreeRate": 0,
      "rateApplied": 0.05,
      "riskFreeRateFloorApplied": true,
      "compoundingFactor": 0.973127050646,
      "dailyCompoundedRate": 0.04865635253,
      "allInRate": -0.95134364747,
      "interestAmount": -264.26,
      "cummulativeInterestAmount": -3377.14
    },
    {
      "interestPeriodDate": "2020-05-12",
      "interestPeriodDays": 1,
      "observationPeriodDate": "2020-05-04",
      "observationPeriodDays": 1,
      "principalBalance": 10000000.0,
      "publishedRiskFreeRate": 0.0659,
      "rateApplied": 0.0659,
      "riskFreeRateFloorApplied": false,
      "compoundingFactor": 0.999660723839,
      "dailyCompoundedRate": 0.0658776417,
      "allInRate": -0.9341223583,
      "interestAmount": -259.48,
      "cummulativeInterestAmount": -3112.88
    },
    {
      "interestPeriodDate": "2020-05-11",
      "interestPeriodDays": 1,
      "observationPeriodDate": "2020-05-01",
      "observationPeriodDays": 3,
      "principalBalance": 10000000.0,
      "publishedRiskFreeRate": 0.0658,
      "rateApplied": 0.0658,
      "riskFreeRateFloorApplied": false,
      "compoundingFactor": 0.982055013743,
      "dailyCompoundedRate": 0.0646192199,
      "allInRate": -0.9353807801,
      "interestAmount": -259.83,
      "cummulativeInterestAmount": -2853.4
    },
    {
      "interestPeriodDate": "2020-05-07",
      "interestPeriodDays": 4,
      "observationPeriodDate": "2020-04-30",
      "observationPeriodDays": 1,
      "principalBalance": 10000000.0,
      "publishedRiskFreeRate": 0.0666,
      "rateApplied": 0.0666,
      "riskFreeRateFloorApplied": false,
      "compoundingFactor": 0.9967887972,
      "dailyCompoundedRate": 0.06638613389,
      "allInRate": -0.93361386611,
      "interestAmount": -1037.35,
      "cummulativeInterestAmount": -2593.57
    },
    {
      "interestPeriodDate": "2020-05-06",
      "interestPeriodDays": 1,
      "observationPeriodDate": "2020-04-29",
      "observationPeriodDays": 1,
      "principalBalance": 10000000.0,
      "publishedRiskFreeRate": 0.0673,
      "rateApplied": 0.0673,
      "riskFreeRateFloorApplied": false,
      "compoundingFactor": 1.000009175023,
      "dailyCompoundedRate": 0.06730061748,
      "allInRate": -0.93269938252,
      "interestAmount": -259.08,
      "cummulativeInterestAmount": -1556.22
    },
    {
      "interestPeriodDate": "2020-05-05",
      "interestPeriodDays": 1,
      "observationPeriodDate": "2020-04-28",
      "observationPeriodDays": 1,
      "principalBalance": 10000000.0,
      "publishedRiskFreeRate": 0.0665,
      "rateApplied": 0.0665,
      "riskFreeRateFloorApplied": false,
      "compoundingFactor": 1.000007327788,
      "dailyCompoundedRate": 0.0665004873,
      "allInRate": -0.9334995127,
      "interestAmount": -259.31,
      "cummulativeInterestAmount": -1297.14
    },
    {
      "interestPeriodDate": "2020-05-04",
      "interestPeriodDays": 1,
      "observationPeriodDate": "2020-04-27",
      "observationPeriodDays": 1,
      "principalBalance": 10000000.0,
      "publishedRiskFreeRate": 0.0652,
      "rateApplied": 0.0652,
      "riskFreeRateFloorApplied": false,
      "compoundingFactor": 1.000005516667,
      "dailyCompoundedRate": 0.06520035969,
      "allInRate": -0.93479964031,
      "interestAmount": -259.67,
      "cummulativeInterestAmount": -1037.83
    },
    {
      "interestPeriodDate": "2020-05-01",
      "interestPeriodDays": 3,
      "observationPeriodDate": "2020-04-24",
      "observationPeriodDays": 3,
      "principalBalance": 10000000.0,
      "publishedRiskFreeRate": 0.0662,
      "rateApplied": 0.0662,
      "riskFreeRateFloorApplied": false,
      "compoundingFactor": 1.0,
      "dailyCompoundedRate": 0.0662,
      "allInRate": -0.9338,
      "interestAmount": -778.17,
      "cummulativeInterestAmount": -778.17
    }
  ]
}