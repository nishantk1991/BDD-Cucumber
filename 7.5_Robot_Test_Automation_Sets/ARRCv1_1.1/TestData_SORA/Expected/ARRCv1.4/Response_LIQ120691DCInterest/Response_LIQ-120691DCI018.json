{
    "comment": " Computed time : 2021-05-21T06:15:20+00:00. Computed time is the time stamp when the calculation was completed.",
    "legalDisclaimer": "The SORA benchmark is used subject to the terms of the UK Open Government License v.3.0 (https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/) and the Bank of England’s disclaimer for SORA (set out below)*. The “SORA” mark is used with the consent of the Bank of England (the benchmark administrator of SORA), and the use of such mark does not imply or express any approval or endorsement by the Bank of England. “Bank of England” and “SORA” are registered trade marks of the Bank of England. The Bank of England has no liability for your use of the calculator service herein.\n*Bank of England’s disclaimer for SORA: SORA is copyright the Bank of England. The trade marks “Bank of England” and “SORA” are registered trade marks of the Bank of England. All Rights Reserved. The SORA benchmark includes the proprietary information of the Bank of England and the data comprising the benchmark may not be copied or distributed except as specifically authorised. The SORA benchmark is not intended to be relied upon as authoritative or taken in substitution for the exercise of judgement. The SORA benchmark is not, and should not be construed as, an offer, bid or solicitation in relation to any financial instrument. The Bank of England does not guarantee, and expressly disclaims any liability for, and makes no representations or warranties whether express or implied, as to the currency, accuracy, timeliness, completeness or fitness for any particular purpose of the SORA benchmark. The Bank of England accepts no liability whatsoever for any loss (including, but not limited to any direct, indirect or consequential loss, whether or not such loss is foreseeable and whether or not the Bank of England has been appraised of the use to which the SORA benchmark will be put) howsoever arising from the use, the timeliness of delivery or the failure of delivery of the SORA benchmark.",
    "interestCalculatorCriteria": {
        "initialPrincipalBalance": 10000000,
        "interestPeriodStartDate": "2020-05-01",
        "interestPeriodEndDate": "2020-06-05",
        "riskFreeRate": "SORA",
        "riskFreeRateFloor": 0.05,
        "lookBackDays": 5,
        "lockoutDays": 0,
        "observationPeriodShift": true,
        "spread": 0,
        "spreadAdjustment": -1,
        "dayCountConvention": "ACT/360",
        "rateRoundingDecimalPoints": 11,
        "paymentRounding": "NEAREST",
        "principalAmountChanges": [
            {
                "date": "2020-05-15",
                "principalAmountChange": 150000
            },
            {
                "date": "2020-05-16",
                "principalAmountChange": -250000
            },
            {
                "date": "2020-05-17",
                "principalAmountChange": -9900000
            }
        ],
        "showDailyDetails": true,
        "calculateTillDate": "2020-05-17"
    },
    "currencyCode": "SGD",
    "interestSummary": [
        {
            "interestPeriodDate": "2020-05-17",
            "interestPeriodDays": 3,
            "observationPeriodDate": "2020-05-07",
            "observationPeriodDays": 4,
            "principalBalance": 0.00,
            "publishedRiskFreeRate": 0,
            "rateApplied": 0.05,
            "riskFreeRateFloorApplied": true,
            "compoundingFactor": 0.909000880856,
            "dailyCompoundedRate": 0.04545004404,
            "allInRate": -0.95454995596,
            "interestAmount": 0.00,
            "cummulativeInterestAmount": -4172.98
        },
        {
            "interestPeriodDate": "2020-05-16",
            "interestPeriodDays": 3,
            "observationPeriodDate": "2020-05-07",
            "observationPeriodDays": 4,
            "principalBalance": 9900000.00,
            "publishedRiskFreeRate": 0,
            "rateApplied": 0.05,
            "riskFreeRateFloorApplied": true,
            "compoundingFactor": 0.909000880856,
            "dailyCompoundedRate": 0.04545004404,
            "allInRate": -0.95454995596,
            "interestAmount": -262.50,
            "cummulativeInterestAmount": -4172.98
        },
        {
            "interestPeriodDate": "2020-05-15",
            "interestPeriodDays": 3,
            "observationPeriodDate": "2020-05-07",
            "observationPeriodDays": 4,
            "principalBalance": 10150000.00,
            "publishedRiskFreeRate": 0,
            "rateApplied": 0.05,
            "riskFreeRateFloorApplied": true,
            "compoundingFactor": 0.909000880856,
            "dailyCompoundedRate": 0.04545004404,
            "allInRate": -0.95454995596,
            "interestAmount": -269.13,
            "cummulativeInterestAmount": -3910.47
        },
        {
            "interestPeriodDate": "2020-05-14",
            "interestPeriodDays": 1,
            "observationPeriodDate": "2020-05-06",
            "observationPeriodDays": 1,
            "principalBalance": 10000000.00,
            "publishedRiskFreeRate": 0.0111,
            "rateApplied": 0.05,
            "riskFreeRateFloorApplied": true,
            "compoundingFactor": 0.977265955303,
            "dailyCompoundedRate": 0.04886329777,
            "allInRate": -0.95113670223,
            "interestAmount": -264.20,
            "cummulativeInterestAmount": -3641.34
        },
        {
            "interestPeriodDate": "2020-05-13",
            "interestPeriodDays": 1,
            "observationPeriodDate": "2020-05-05",
            "observationPeriodDays": 1,
            "principalBalance": 10000000.00,
            "publishedRiskFreeRate": 0,
            "rateApplied": 0.05,
            "riskFreeRateFloorApplied": true,
            "compoundingFactor": 0.973127050646,
            "dailyCompoundedRate": 0.04865635253,
            "allInRate": -0.95134364747,
            "interestAmount": -264.26,
            "cummulativeInterestAmount": -3377.14
        },
        {
            "interestPeriodDate": "2020-05-12",
            "interestPeriodDays": 1,
            "observationPeriodDate": "2020-05-04",
            "observationPeriodDays": 1,
            "principalBalance": 10000000.00,
            "publishedRiskFreeRate": 0.0659,
            "rateApplied": 0.0659,
            "riskFreeRateFloorApplied": false,
            "compoundingFactor": 0.999660723839,
            "dailyCompoundedRate": 0.06587764170,
            "allInRate": -0.93412235830,
            "interestAmount": -259.48,
            "cummulativeInterestAmount": -3112.88
        },
        {
            "interestPeriodDate": "2020-05-11",
            "interestPeriodDays": 1,
            "observationPeriodDate": "2020-05-01",
            "observationPeriodDays": 3,
            "principalBalance": 10000000.00,
            "publishedRiskFreeRate": 0.0658,
            "rateApplied": 0.0658,
            "riskFreeRateFloorApplied": false,
            "compoundingFactor": 0.982055013743,
            "dailyCompoundedRate": 0.06461921990,
            "allInRate": -0.93538078010,
            "interestAmount": -259.83,
            "cummulativeInterestAmount": -2853.40
        },
        {
            "interestPeriodDate": "2020-05-07",
            "interestPeriodDays": 4,
            "observationPeriodDate": "2020-04-30",
            "observationPeriodDays": 1,
            "principalBalance": 10000000.00,
            "publishedRiskFreeRate": 0.0666,
            "rateApplied": 0.0666,
            "riskFreeRateFloorApplied": false,
            "compoundingFactor": 0.996788797200,
            "dailyCompoundedRate": 0.06638613389,
            "allInRate": -0.93361386611,
            "interestAmount": -1037.35,
            "cummulativeInterestAmount": -2593.57
        },
        {
            "interestPeriodDate": "2020-05-06",
            "interestPeriodDays": 1,
            "observationPeriodDate": "2020-04-29",
            "observationPeriodDays": 1,
            "principalBalance": 10000000.00,
            "publishedRiskFreeRate": 0.0673,
            "rateApplied": 0.0673,
            "riskFreeRateFloorApplied": false,
            "compoundingFactor": 1.000009175023,
            "dailyCompoundedRate": 0.06730061748,
            "allInRate": -0.93269938252,
            "interestAmount": -259.08,
            "cummulativeInterestAmount": -1556.22
        },
        {
            "interestPeriodDate": "2020-05-05",
            "interestPeriodDays": 1,
            "observationPeriodDate": "2020-04-28",
            "observationPeriodDays": 1,
            "principalBalance": 10000000.00,
            "publishedRiskFreeRate": 0.0665,
            "rateApplied": 0.0665,
            "riskFreeRateFloorApplied": false,
            "compoundingFactor": 1.000007327788,
            "dailyCompoundedRate": 0.06650048730,
            "allInRate": -0.93349951270,
            "interestAmount": -259.31,
            "cummulativeInterestAmount": -1297.14
        },
        {
            "interestPeriodDate": "2020-05-04",
            "interestPeriodDays": 1,
            "observationPeriodDate": "2020-04-27",
            "observationPeriodDays": 1,
            "principalBalance": 10000000.00,
            "publishedRiskFreeRate": 0.0652,
            "rateApplied": 0.0652,
            "riskFreeRateFloorApplied": false,
            "compoundingFactor": 1.000005516667,
            "dailyCompoundedRate": 0.06520035969,
            "allInRate": -0.93479964031,
            "interestAmount": -259.67,
            "cummulativeInterestAmount": -1037.83
        },
        {
            "interestPeriodDate": "2020-05-01",
            "interestPeriodDays": 3,
            "observationPeriodDate": "2020-04-24",
            "observationPeriodDays": 3,
            "principalBalance": 10000000.00,
            "publishedRiskFreeRate": 0.0662,
            "rateApplied": 0.0662,
            "riskFreeRateFloorApplied": false,
            "compoundingFactor": 1.000000000000,
            "dailyCompoundedRate": 0.06620000000,
            "allInRate": -0.93380000000,
            "interestAmount": -778.17,
            "cummulativeInterestAmount": -778.17
        }
    ]
}